docker-compose.yml

```diff
@@ -9,16 +9,36 @@ services:
       POSTGRES_USER: wordle_user
     volumes: 
       - dbdata:/var/lib/postgresql/data
-  # server:
-  #   build:
-  #     context: wordle
-  #     dockerfile: Dockerfile
-  #   environment:
-  #     WHALEAPP__DB_HOST: db
-  #     WHALEAPP__DB_NAME: wordle_db
-  #     WHALEAPP__DB_USER: wordle_user
-  #     WHALEAPP__DB_PASSWORD: wordle_password
 
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready"]
+      interval: 10s
+      timeout: 5s
+      retries: 20
+
+  app:
+    build:
+      context: ~/Desktop/jump/projects/wordle/frontend
```

Comment:
Never use absolute paths. If you do this anyone who wants to use your project has to store it in the same directory you mention here. `frontend` is enough, assuming people will run Docker Compose from the root (specify in the README if you haven't already). Same thin for the server down below

Reply:
Thank you Leo, as you said it's from the root, should I include the ~? 

I.e. have it as `~/frontend? `

I have updated the ReadMe file so it's a lot more tidier. Also I was considering keeping a log of all the errors I was getting (basically what was on the read me before). Is that worthwhile?

Reply:
Nope, `~` is not the root, it's the home directory, the root is `/`. However, I meant the root of the project (I didn't make myself clear), so you just need to specify `frontend` and Docker Compose will look into the directory with that name contained in the current directory.